### PR TITLE
1.3 misc changes

### PR DESCRIPTION
**Pull request by issa-tseng**
still to include maybe a couple more bug patches, and final??? security updates.

### CODE REVIEW

File: lib/util/problem.js [hunk 1]
```diff
@@ -179,15 +179,15 @@ const problems = {
     unsupportedODataField: problem(501.5, (({ at, text }) => `The given OData filter expression references fields not supported by this server: ${text} at ${at}`)),
     unsupportedODataExpandExpression: problem(501.6, (({ text }) => `The given OData expand expression is not supported by this server: "${text}". Currently, only "$expand=*" is supported.`)),
 
+    // returned in case no Enketo service is configured.
+    enketoNotConfigured: problem(501.7, () => 'This ODK Central has not been configured to use Enketo. Please contact your server administrator.'),
```
Comment (matthew-white) [inline lib/util/problem.js:183]: I actually just added 501.7 for `invalidDatabaseConfig` in #394.

**Review by matthew-white — COMMENTED** [review]
> @matthew-white can you look over my work here? [086a59b](https://github.com/getodk/central-backend/commit/086a59b9f583a18d599911344991c347c4f0c582)
> the rfcs are very very very very confusing.

Definitely agreed that the RFCs are very confusing. (Especially the fact that the single quotes are required, but the language isn't, resulting in the fun prefix `UTF-8''`.) I didn't take a look at tests, but the implementation here looks correct. 💯

**Comment by matthew-white** [comment]
Unrelatedly, I'm noticing that there seems to be a merge conflict (I think related again to the Problem codes).

File: lib/util/http.js [hunk 2]
```diff
@@ -69,9 +69,35 @@ const xml = contentType('application/xml');
...
+    .replace(/\*/g, '%2A')
+    // The following are not required for percent-encoding per RFC5987,
+    // so we can allow for a little better readability over the wire: |`^
+    .replace(/%(?:7C|60|5E)/g, unescape);
+
+const sanitizeToAscii = (str) => str.replace(/[^\x00-\x7F]+/g, '-'); //  eslint-disable-line no-control-regex
+
+const contentDisposition = (filename) => {
+  const sanitized = sanitize(filename);
+  return `attachment; filename="${sanitizeToAscii(sanitized)}" filename*=UTF-8''${encodeRFC5987ValueChars(sanitized)}`;
```
[the remark this answers is not in the document]
Comment (issa-tseng) [reply lib/util/http.js:92]: oh right. i can just adjust the regex range to  exclude those chars i  think?

[the remark this answers is not in the document]
Comment (issa-tseng) [reply lib/util/http.js:92]: oh right slashes  etc

[the remark this answers is not in the document]
Comment (issa-tseng) [reply lib/util/http.js:92]: oh you're saying it's already probably okay

[the remark this answers is not in the document]
Comment (matthew-white) [reply lib/util/http.js:92]: Yeah, `sanitize()` would hopefully take care of the cases like that. 🤔

**Review by matthew-white — APPROVED** [review]

**Pull request merged**